src/Blubrry/REST/Resource/PodcastStatistics.php

```diff
@@ -42,21 +42,21 @@ public function summary($program_keyword, $month = null, $year = null) {
      *
      * @since 1.0.0
      *
-     * @param string $program_keyword
+     * @param string $programKeyword
      * @param array $params
      *
      * @return array The API response.
      */
-    public function totals($program_keyword, $params) {
-        $path = '/2/stats/' . $program_keyword . '/totals.json';
-        $qs='?';
+    public function totals($programKeyword, $params) {
+        $path = '/2/stats/' . $programKeyword . '/totals.json';
+        $qs   = '?';
 
         if (empty($params)) {
             return false;
         }
     
         foreach ($params as $key => $item) {
```

Comment:
@lucashillebrandt Here I believe we could replace the foreach using the `http_build_query` function from PHP, right?

Reply:
Yes, that's right, this one is now fixed